tests/scripts/protect-i18n.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+/**
+ * What is this? With this small JavaScript file we want
+ * to block the upload of modified files in the "locale"
+ * folder to the repository through a git hook generated
+ * with the Husky library.
+ *
+ * Usage: $ node .git-hook/locale.js
+ *
+ * Happy releasing!
+ */
+
+'use strict'
+
+// Require main dependences.
+const { execSync } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+// Execute Git Diff for get all files with changes.
+const results = execSync('git diff --cached --name-only')
+
+// Create array with all files.
+const arrayResults = results.toString().trim().split('\n')
+
+// Define array files to not change.
+const changedLocalizedFiles = []
+
+function gitHook () {
+  // Define RegExp with reg param.
+  const reg = new RegExp('/locale/', 'g')
```

Comment:
```suggestion
  const reg = /locale\/(?!en)/g;
```
We should allow original content changes, so `en` locale should be excluded from protection